datapreview.tex

```diff
@@ -0,0 +1,126 @@
+DP0.1 only uses the 300~\sqdeg of DC2 images that were simulated for five years of the LSST’s wide-fast-deep component (WFD) using a baseline cadence, \citedsp{PSTN-055}.
+The DESC processed the simulated DC2 images with \href{https://pipelines.lsst.io/v/v19_0_0/index.html}{Version 19} of the LSST Science Pipelines. 
+DP0.1 makes the DESC’s DC2 images and catalogs available to users through an early version the Rubin Science Platform (RSP) running at the US DAC. 
+
+For full details on DP0.1 including an exact description of the data products served, see the documentation at \url{https://dp0-1.lsst.io/}
+
+\subsubsection{Data Preview 0.2}
+
+Data Preview 0.2 (DP0.2) was released to approximately 600 early adopters from the community in June 2022, exactly 1 year after DP0.1. 
+The dataset used for DP0.2 was the same as that used for DP0.1.
+Rubin processed the simulated DC2 images with \href{https://pipelines.lsst.io/v/v23_0_0/index.html}{Version 23} of the LSST Science Pipelines. 
+DP0.2 makes the Rubin reprocessed DESC DC2 images and catalogs available to users through an early version the Rubin Science Platform (RSP) running at the US DAC. 
+
+For full details on DP0.2 including an exact description of the data products served, see the documentation at \url{https://dp0-2.lsst.io/}
+
+\subsubsection{Data Preview 0.3}
+
+Scheduled for between June and September 2023, DP0.3 will be the last in the DP0 series of Data Previews based on simulated LSST-like data. 
+The main goal of DP0.3 is to support the Solar System Science Collaboration by hosting their simulated 10-year catalog and serving it via the RSP at the US DAC. 
+Table~\ref{tab:dp-zero-three-products} presents a summary of the expected DP0.3 data products, as of January 2023.
+The final exact data products for DP0.3 are still to be decided. 
+
+\subsection{Data Preview 1}
+
+Data Preview 1 was originally defined to be based on reprocessed on-sky data taken with ComCam.
+Following the replan of the Construction project in December 2022, no on-sky data will be taken with ComCam, \S~\ref{sec:commissioning}.
+Consequently DP1 has been redefined to be based on a subset of science-grade images taken with LSSTCam during a period of a few days around the System First Light milestone, \S~\ref{ssec:commissioning-observations}.
+The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
+At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects. 
+
+Note that the DP1 period of time during which data for DP1 are collected is \textit{very short}: the data products released in DP1 will be generated from relatively few observations taken in the few days around System First Light.
+Table~\ref{tab:dp-one-products} presents a summary of the data products expected in DP1, as of January 2023.
+
+Data Preview 1 is currently expected between September to October 2024. 
```

Comment:
```suggestion
```

Reply:
Suggest we leave the DP1 date for the timeline section, so we don't have to update the date ranges in two places?